internal/testharness/pty/blackbox/environment.go

```
@@ -0,0 +1,423 @@
+	}
+	return fmt.Errorf("standalone server readiness timed out: stderr=%s", e.Server.stderr.String())
+}
+
+func (e *IsolatedEnvironment) BindProject() error {
+	dialContext, cancelDial := context.WithTimeout(context.Background(), fixedWait)
+	remote, err := client.DialRemoteURL(dialContext, "ws://"+net.JoinHostPort(e.Host, strconv.Itoa(e.Port))+"/rpc")
+	cancelDial()
+	if err != nil {
+		return fmt.Errorf("dial standalone server project API: %w", err)
+	}
+	defer remote.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `remote.Close` is not checked

(errcheck)